src/test/resources/export/setup.sql

```diff
@@ -2,15 +2,15 @@ INSERT INTO audit_event
     (id, uuid, case_uuid, stage_uuid, correlation_id, raising_service, audit_payload, namespace, audit_timestamp, type, user_id, case_type)
 VALUES (0, '00000000-0000-0000-0000-000000000000', '10000000-0000-0000-0000-000000000000',
```

Comment:
For non-CMS cases migratedReference will be null, can a test case where this is null/undefined be added